test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -571,12 +662,26 @@ class AccountListPage {
     console.log(
       `Verify the number of accounts in the account menu is: ${expectedNumberOfAccounts}`,
     );
-    await this.driver.wait(async () => {
-      const internalAccounts = await this.driver.findElements(
-        this.accountListItem,
-      );
-      return internalAccounts.length === expectedNumberOfAccounts;
-    }, 20000);
+
+    await this.driver.waitForSelector(this.accountListItem);
+    await this.driver.wait(
+      async () => {
+        await this.driver.delay(500);
+        const internalAccounts = await this.driver.findElements(
+          this.accountListItem,
+        );
+        console.log(
+          `Number of accounts: ${
+            internalAccounts.length
+          } is equal to ${expectedNumberOfAccounts}? ${
+            internalAccounts.length === expectedNumberOfAccounts
+          }`,
+        );
+        return internalAccounts.length === expectedNumberOfAccounts;
```

Comment:
Nit: Just to be more DRY:

```suggestion
        const isValid = internalAccounts.length === expectedNumberOfAccounts;
        console.log(
          `Number of accounts: ${
            internalAccounts.length
          } is equal to ${expectedNumberOfAccounts}? ${
            isValid
          }`,
        );
        return isValid;
```